estate/models/estate_property.py

```diff
@@ -45,4 +46,15 @@ class EstateProperty(models.Model):
         required=True,
         copy=False,
     )
-    active = fields.Boolean("Active", default=True)
+    property_type_id = fields.Many2one("estate_property_type", "Property Type")
+    salesman_id = fields.Many2one(
+        "res.users",
+        "Salesman",
+        default=lambda self: self.env.user,
+    )
```

Comment:
It's ok to use this way of indent, but sometimes it's not necessary. A one-line would be ok here